src/services/authService.js

```diff
@@ -50,4 +50,19 @@ const signUp = async (email, password, firstName, lastName) => {
     }
 }
 
-export {signOut, signIn, signUp}
\ No newline at end of file
+/**
+ * A "get" request to the firebase cloud firestore which retrieves the first and last name for a specific user
+ * @param {*} userId Unique user identifier as a string. You can get this from currentUser.uid
+ * @returns A single javascript object with the following fields:
+ * First_name: string,
+ * Last_name: string,
+ * User_id: string
+ */
+ const fetchUserInfo = async (userId) => {
```

Comment:
imo this should be done within the authContext provider. If it sees that the auth context has just changed, and has a user associated, it could call this method and load the first and last name into the authContext. That way it's accessible from anywhere in the app, without having to make extra db calls in every page that wants to use it

Reply:
Actually this is a good point.
Thinking about it though, would we want other context variables or put it into the currentUser that is present (don't know if it's possible but should be??)

Reply:
I think ideally it would be a ```user``` object with something like user.id, user.email, user.first_name, user.last_name. You might need to be a custom user object, and just user the values from the one that firebase is giving you

Reply:
sounds like a good idea, I'll try implement it